src/equations/compressible_euler_3d.jl

```diff
@@ -342,12 +349,13 @@ Should be used together with [`TreeMesh`](@ref).
                                               surface_flux_function,
                                               equations::CompressibleEulerEquations3D)
     # get the appropriate normal vector from the orientation
+    RealT = eltype(u_inner)
     if orientation == 1
-        normal_direction = SVector(1.0, 0.0, 0.0)
+        normal_direction = SVector(one(RealT), 0, 0)
```

Comment:
Again, this seems odd to me. Why only convert one of the values?

Reply:
See below